tests/integration/test_cli.py

```diff
@@ -20,24 +20,24 @@ def runner():
     return CliRunner()
 
 
-def test_pipeline_adds_request(runner, mongo_port, cfg, apache_req):
+def test_pipeline_adds_request(runner, mongo_uri, cfg, apache_req):
     with open(cfg) as fp:
         config = yaml.load(fp)
-    config['MONGO_CONNECTION'] = 'mongodb://localhost:%d' % mongo_port
+    config['MONGO_CONNECTION'] = [mongo_uri]
```

Comment:
This will clobber bdd70b65fcfa5a87018fd7810a9a801dc01c0faa. Just replace it with:
```python
config['MONGO_CONNECTION'] = 'mongodb://%s' % mongo_uri
```
One more case down in line 40.